app/uk/gov/hmrc/cataloguefrontend/service/ConfigService.scala

```diff
@@ -43,6 +43,7 @@ object ConfigService {
   case class ConfigSourceEntries(source: String, precedence: Int, entries: Map[KeyName, String] = Map())
   case class ConfigSourceValue(source: String, precedence: Int, value: String)
 
+  // TODO another variant to cater for - Environment + "local"...
```

Comment:
What does this comment mean?

Reply:
I have further work to add Environment + Latest, and Environment + Latest + GithubMaster types. This is another case I don't want to loose.